packages/picasso/src/Dropdown/styles.ts

```diff
@@ -51,4 +59,8 @@ export default ({ screens, shadows, palette }: Theme) =>
         maxHeight: '100vh',
       },
     },
+    contentStyle: {
+      height: ({ contentStyle }: StyleProps) => contentStyle?.height,
+      maxHeight: ({ contentStyle }: StyleProps) => contentStyle?.maxHeight,
+    },
```

Comment:
Do you need two separate functions?
```suggestion
    contentStyle: ({ contentStyle }: StyleProps) => ({
      height: contentStyle?.height,
      maxHeight: contentStyle?.maxHeight,
    }),
```
or even:
```suggestion
    contentStyle: ({ contentStyle }: StyleProps) => ({
      ...contentStyle
    }),
```

Reply:
way better your options, thanks man 😄